spark/src/main/scala/org/apache/comet/testing/ParquetGenerator.scala

```diff
@@ -205,6 +212,11 @@ object ParquetGenerator {
         Range(0, numRows).map(_ => new java.sql.Date(1716645600011L + r.nextInt()))
       case DataTypes.TimestampType =>
         Range(0, numRows).map(_ => new Timestamp(1716645600011L + r.nextInt()))
+      case DataTypes.TimestampNTZType =>
```

Comment:
Do we know what this magic number is? If so could we define it as a variable?

Reply:
Updated. It was approximately the current date when I worked on this code. I added a variable with comments.

Reply:
Or you could just use `SimpleDateFormat("YYYY-MM-DD hh:mm:ss").parse("1980-12-08 23:15:00")` ?

Reply:
Updated